components/dashboards-web-component/src/designer/DashboardDesigner.css

```diff
@@ -92,4 +92,8 @@
 
 .publishers-list {
     margin-top: 10px;
+}
+
+.home-page-checkbox {
+    padding: 30px 0px;
 }
```

Comment:
Newline at the end of file is missing. Please add a new line on end of the files.

Reply:
Outdated, file not found in master